WebFileSystem.ts

```diff
@@ -613,12 +613,12 @@ class WebFileSystem extends webdav.FileSystem {
                 if (await this.loadPath(path, user)) {
                     callback(null, this.resources.get(user.uid).get(path.toString()).type)
                 } else {
-                    logger.error('File could not be found!')
+                    logger.error(`WebFileSystem._type : File could not be found! uid: ${info.context.user.uid} path: ${path.toString()}`, new Error('Stack-Tracer'))
                     callback(webdav.Errors.ResourceNotFound)
                 }
             }
         } else {
-            logger.error(webdav.Errors.BadAuthentication.message)
+            logger.error(`WebFileSystem._type : ${webdav.Errors.BadAuthentication.message} path: ${path.toString()}`, new Error('Stack-Tracer'))
```

Comment:
`warn` or `debug`? Also the other messages contained `context.user.false`, this one leaves it out.